dgraphtest/cluster.go

```diff
@@ -62,8 +62,8 @@ type HttpToken struct {
 // HTTPClient allows doing operations on Dgraph over http
 type HTTPClient struct {
 	*HttpToken
-	adminURL   string
-	graphqlURL string
+	AdminURL   string
```

Comment:
instead of exposing this, create a function in dgraphtest package to give you a HTTPClient object with args that you have.